src/main/java/core/basesyntax/serviceimpl/DataConverterImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataConverterImpl implements DataConverter<List<String>, List<FruitTransaction>> {
```

Comment:
The class name 'DataConverterImpl' could be more descriptive. Consider renaming it to reflect the specific type of data it is converting, such as 'StringToTransactionConverter'.

Reply:
Implementation class name could be a bit more descriptive